scripts/generate/append-operator-version.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
I'd suggest use the label as we have on the managed resources: `operator.prometheus.io`
WDYT?

Reply:
Hmm, makes sense! 👍🏼